example/src/App.tsx

```diff
@@ -1,45 +1,18 @@
-import { Typo } from '@components/Typo';
-import { SERVER_URL } from '@env';
-import * as Membrane from '@jellyfish-dev/react-native-membrane-webrtc';
+import { NavigationContainer } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
 import { useFonts } from 'expo-font';
 import * as SplashScreen from 'expo-splash-screen';
-import React, { useCallback, useEffect, useState } from 'react';
-import {
-  StyleSheet,
-  View,
-  PermissionsAndroid,
-  SafeAreaView,
-  Text,
-  TextInput,
-  Platform,
-  Pressable,
-  Switch,
-  Alert,
-} from 'react-native';
+import React, { useEffect } from 'react';
 
+import { Home } from './Home';
```

Comment:
Consider putting screens in a separate folder `screens` and maybe importing them like `@screens`? We're going to have a bit more of screens in the future.

Reply:
Done.